### PR TITLE
remove unnecessary pylint ignores

### LINKED ISSUE #828: Unnecessary invalid-name linting directives on tests
The merge of #724 has reintroduced the `pylint: disable=invalid-name` directive removed in #739

### PR DESCRIPTION
**Pull request by ajavadia**
### Summary
Fix #828

### CODE REVIEW

File: test/python/test_circuit.py [hunk 1]
```diff
@@ -48,8 +48,8 @@ def test_circuit_qasm(self):
         """
         qr1 = QuantumRegister(1, 'qr1')
         qr2 = QuantumRegister(2, 'qr2')
-        c = ClassicalRegister(3, 'c')
-        qc = QuantumCircuit(qr1, qr2, c)
+        cr = ClassicalRegister(3, 'cr')
```
Comment (delapuente) [inline test/python/test_circuit.py]: There is a test failing, something related to changing the internal names of registers and then comparing with some "expected qasm". You can preserve the internal name of the register or fix the test accordingly.

**Comment by ajavadia** [comment]
@delapuente missed that one, it should be fixed now.

**Comment by ajavadia** [comment]
@delapuente good catch. done.

File: test/python/test_identifiers.py [hunk 2]
```diff
@@ -42,44 +33,44 @@ def setUp(self):
...
-
-    @unittest.skipIf(_skip_cpp, "no c++ simulator found.")
+        exp = qobj.experiments[0]
+        c_qasm = exp.header.compiled_circuit_qasm
+        self.assertIn(self.qr_name, map(lambda x: x[0], exp.header.qubit_labels))
+        self.assertIn(self.qr_name, c_qasm)
+        self.assertIn(self.cr_name, map(lambda x: x[0], exp.header.clbit_labels))
+        self.assertIn(self.cr_name, c_qasm)
+
+    @requires_cpp_simulator
```
Comment (delapuente) [inline test/python/test_identifiers.py]: Nice catch! :tada:

**Pull request merged**